convex/httpUtils.ts

```
@@ -0,0 +1,133 @@
+import { httpAction } from './_generated/server';
+import { createClient } from '@supabase/supabase-js';
+
+// Initialize Supabase client for user verification
+const supabase = createClient(
+  process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)